beetle_battle.py

```diff
@@ -5,30 +5,53 @@
 import arcade
 
 # Constants
-SCREEN_WIDTH = 1024
+SCREEN_WIDTH = 1280
 SCREEN_HEIGHT = 720
 SCREEN_TITLE = "Welcome to Arcade"
 
+# Scaling constants
+BEETLE_SCALING = .65
```

Comment:
I don't know if I see a benefit to separating this out from the other constants just yet. Usually if you're going to make a grouping like this and you only end up having one or two lines, it's probably too small to justify a grouping.

Reply:
My thinking is that there could be other sprites and actors we could add down the road (elites, queens, larvae, also body parts like mandibles, projectiles) and maybe during testing we decide things need to be bigger or smaller. I'm assuming this could allow us to scale the sprite and hitbox in one easy spot. Thoughts?

Reply:
Yes, and that's good thinking! However, one of the things I've learned is that it often makes sense to save doing work until you have proven you _need_ to do the work. So in this case, as written we don't need it. Then when we _do_ start adding more scaling constants, we can add the separate grouping _then_.

This is to avoid a common pattern of early optimization or organization that ends up never needing being needed.

But really, this is a small enough thing I don't feel _so_ strongly about, feel free to resolve this "won't fix".

Reply:
Gotcha! So what you're saying is

![image](https://user-images.githubusercontent.com/4132786/229636131-7ed4ed5c-6afa-4ee0-a66b-2cdf87cd6d81.png)

Reply:
EXACTLY